template/FullStack/Vue(Frontend)+Django(Backend)/server/api/views/user_view.py

```
@@ -0,0 +1,20 @@
+from rest_framework import status
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework.permissions import IsAuthenticated
+from rest_framework_simplejwt.authentication import JWTAuthentication
+from ..serializers import UserSerializer
```

Ruff

`..serializers.UserSerializer` imported but unused

Remove unused import: `..serializers.UserSerializer`

(F401)